### DISCUSSION TITLE
Confirm shipment calls not available?

### DISCUSSION THREAD

**Discussion by johncarlson21** [amazon-php/sp-api-sdk#411] [Q&A]
Are the calls to confirm shipment not part of this library? 

I need to be able to push up my tracking numbers but I am not seeing where this is handled. I see it in the api docs but not in the library.

**Comment by sergisolanellas** [comment]
To send the tracking numbers, you must use the feedapi.

**Reply by johncarlson21** [reply]
do you have any type of example? or what calls to make?

**Reply by johncarlson21** [reply]
I see that it is part of the order model in the selling partner api models:

"/orders/v0/orders/{orderId}/shipmentConfirmation": {
      "post": {
        "tags": [
          "ordersV0"
        ],
        "description": "Updates the shipment confirmation status for a specified order.\n\n**Usage Plan:**\n\n| Rate (requests per second) | Burst |\n| ---- | ---- |\n| 2 | 10 |\n\nThe `x-amzn-RateLimit-Limit` response header returns the usage plan rate limits that were applied to the requested operation, when available. The table above indicates the default rate and burst values for this operation. Selling partners whose business demands require higher throughput may see higher rate and burst values then those shown here. For more information, see [Usage Plans and Rate Limits in the Selling Partner API](doc:usage-plans-and-rate-limits-in-the-sp-api).",
        "operationId": "confirmShipment",
        "parameters": [
          {
            "name": "orderId",
            "in": "path",
            "description": "An Amazon-defined order identifier, in 3-7-7 format.",
            "required": true,
            "type": "string"
          },
          {
            "name": "payload",
            "in": "body",
            "description": "Request body of confirmShipment.",
            "required": true,
            "schema": {
              "$ref": "#/definitions/ConfirmShipmentRequest"
            }
          }
        ],

**Comment by johncarlson21** [comment]
So apparently, when I required the library into my project it only pulled the 4.x version.. 5.x version has the call
